rules/shortsubject/shortsubject.go

```diff
@@ -21,11 +23,12 @@ func init() {
 // ValidateShortSubject checks that the commit's subject is strictly less than
 // 90 characters (preferably not more than 72 chars).
 func ValidateShortSubject(c git.CommitEntry) (vr validate.Result) {
-	if len(c["subject"]) >= 90 {
+	if len(c["subject"]) >= 90 && len(strings.Split(c["parent"], " ")) <= 1 {
 		vr.Pass = false
 		vr.Msg = "commit subject exceeds 90 characters"
 		return
 	}
+
 	vr.Pass = true
 	if len(c["subject"]) > 72 {
 		vr.Msg = "commit subject is under 90 characters, but is still more than 72 chars"
```

Comment:
this is no longer necessary true. could do with another conditional branch dealing with the new case